packages/react/dismissable-layer/src/DismissableLayer.stories.tsx

```diff
@@ -643,3 +644,23 @@ function DummyPopover({
     </Popper.Root>
   );
 }
+
+export const InPopupWindow = () => {
```

Comment:
nit: can we move this just above the "Chromatic" story, as we like to keep this one last. Thanks!

Reply:
`DismissableLayer` does not appear to have a `Chromatic` story.